src/videotile/DefaultVideoTile.ts

```diff
@@ -46,6 +47,12 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
     if (videoElement.srcObject !== videoStream) {
       videoElement.srcObject = videoStream;
     }
+
+    if (DefaultVideoTile.isSafariBrowser()) {
```

Comment:
Instead of using browser detection, let's wrap this up in a behavior. Add a behavior to BrowserBehavior (e.g. 'requiresManualVideoPlay') and have that use the BrowserBehavior private method for detecting Safari.